plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -23,6 +23,799 @@ hub_tags:
+      title: "Os Actor Process Image Name"
+      type: string
+      description: "Os actor process image name"
+      required: false
+    os_actor_process_image_path:
+      title: "Os Actor Process Image Path"
+      type: string
+      description: "Os actor process image path"
+      required: false
+    os_actor_process_image_sha256:
+      title: "Os Actor Process Image SHA256"
+      type: string
+      description: "Os actor process image SHA256"
+      required: false
+    os_actor_process_instance_id:
+      title: "Os Actor Process Instance ID"
+      type: string
+      description: "Os actor process instance ID"
+      required: false
+    os_actor_process_os_pid:
+      title: "Os Actor Process Os PID"
+      type: string
+      description: "Os actor process os PID"
+      required: false
+    os_actor_process_signature_status:
+      title: "Os Actor Process Signature Status"
+      type: string
+      description: "Os actor process signature status"
+      required: false
+    os_actor_process_signature_vendor:
+      title: "Os Actor Process Signature Vendor"
+      type: string
+      description: "Os actor process signature vendor"
+      required: false
+    os_actor_thread_thread_id:
+      title: "Os Actor Thread Thread ID"
+      type: string
+      description: "Os actor thread thread ID"
+      required: false
+    story_id:
+      title: "Story ID"
+      type: string
+      description: "Story ID"
+      required: false
+    user_name:
+      title: "User Name"
+      type: string
+      description: "User name"
+      required: false
+  alert:
+    action:
+      title: "Action"
+      type: string
+      description: "Action"
+      required: false
+    action_pretty:
+      title: "Action Pretty"
+      type: string
+      description: "Action pretty"
+      required: false
+    agent_data_collection_status:
+      title: "Agent Data Collection Status"
+      type: boolean
+      description: "Agent data collection status"
+      required: false
+    agent_device_domain:
+      title: "Agent Device Domain"
+      type: string
+      description: "Agent device domain"
+      required: false
+    agent_fqdn:
+      title: "Agent FQDN"
+      type: string
+      description: "Agent FQDN"
+      required: false
+    agent_os_sub_type:
+      title: "Agent Os Sub Type"
+      type: string
+      description: "Agent os sub type"
+      required: false
+    agent_os_type:
+      title: "Agent Os Type"
+      type: string
+      description: "Agent os type"
+      required: false
+    agent_version:
+      title: "Agent Version"
+      type: string
+      description: "Agent version"
+      required: false
+    alert_id:
+      title: "Alert ID"
+      type: string
+      description: "Alert ID"
+      required: false
+    attempt_counter:
+      title: "Attempt Counter"
+      type: string
+      description: "Attempt counter"
+      required: false
+    bioc_category_enum_key:
+      title: "Bioc Category Enum Key"
```

Comment:
BIOC should be capitalized everywhere (it's an acronym for Behavior Indicator of Compromise)